src/LiveComponent/doc/index.rst

```diff
@@ -1326,6 +1326,51 @@ The files will be available in a regular ``$request->files`` files bag::
     need to specify ``multiple`` attribute on HTML element and end ``name``
     with ``[]``.
 
+.. _downloads:
+
+Downloading files
+-----------------
+
+Currently, Live Components do not natively support returning file responses directly from a LiveAction. However, you can implement file downloads by redirecting to a route that handles the file response.
+
+Create a LiveAction that generates the URL for the file download and returns a `RedirectResponse`::
+
+    use Symfony\Component\HttpFoundation\RedirectResponse;
+    use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+    use Symfony\UX\LiveComponent\Attribute\LiveAction;
+
+    // ...
+
+    class MyDownloadButton
+    {
+    // ...
+
+        #[LiveAction]
+        public function initiateDownload(UrlGeneratorInterface $urlGenerator): RedirectResponse
+        {
+            $url = $urlGenerator->generate('app_file_download');
+            return new RedirectResponse($url);
+        }
+    }
+
+.. code-block:: html+twig
+
+    <div {{ attributes }} data-turbo="false">
+        <button
+            data-action="live#action"
+            data-live-action-param="initiateDownload"
+        >
+        Download
+        </button>
+    </div>
+
+When Turbo is enabled, it will intercept the redirect and initiate a second request for the download URL. Adding `data-turbo="false"` ensures that the download URL is called only once.
+
+.. note::
+
+   Native support for file downloads in Live Components is under development. For more details, refer to the related `pull request #2483 <https://github.com/symfony/ux/pull/2483>`_.
+
```

Comment:
```suggestion
   Native support for file downloads in Live Components is under development. For more details, refer to the related `pull request #2483 <https://github.com/symfony/ux/pull/2483>`_.

```

Let's not document what's not yet released, to avoid any frustration :)